.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -18,31 +18,44 @@ For OEM releases, keep the OEM Release template and remove the Open Release one
 
  - [ ] [DOC] Ping in #documentation-internal about the new release
  - [ ] [GIT] Create branch `release/M.m.p` in owncloud/android from master
+ - [ ] [GIT] Rebase `release/M.m.p` against `stable` in owncloud/android
+ - [ ] [GIT] Create branch `release/x.y.z` in owncloud/android-library from master
+ - [ ] [GIT] Rebase `release/x.y.z` against `stable` in owncloud/android-library
  - [ ] [DEV] Update version number and name in build.gradle in owncloudApp module
  - [ ] [DEV] Update [SBOM](https://cloud.owncloud.com/f/6072870)
  - [ ] [DIS] Create a folder for the new version like `M.m.p_YYYY-MM-DD` inside the `changelog` folder
  - [ ] [DIS] Move all changelog files from the unreleased folder to the new version folder
  - [ ] [DIS] Update screenshots, if needed, in README.md
  - [ ] [DIS] Add ReleaseNotes replacing `emptyList` with `listOf` and adding inside `ReleaseNote()` with String resources
+ - [ ] [DIS] Prepare post in central.owncloud.org ([`Category:News + Tag:android`](https://central.owncloud.org/tags/c/news/5/android))
  - [ ] [QA] Design Test plan
  - [ ] [QA] Regression Test execution
- - [ ] [GIT] Create and sign tag `oc-android-M.m.p` in HEAD commit of release branch, in owncloud/android
- - [ ] [GIT] Create and sign tag `x.y.z` in HEAD commit of release branch, in owncloud/android-library
  - [ ] [DIS] Generate final bundle from signed commit in owncloud/android
```

Comment:
I think this step should go before `[QA] Regression Test execution` since this bundle is used in QA and if everything is OK, we use this same bundle to upload to Google Play. If there's something to change, I think the generation of a new bundle is implicit in the `[QA] Regression Test execution` step, otherwise we should add another additional step specifying this or add it in that step like `[QA] Regression Test execution and bugfixing`.

Reply:
The final approval is implicit in the QA stage, so, it can go before. During QA, bugfixing will trigger another bundle creation and then, more QA. Will change.